.github/actions/setup-java-build/action.yml

```
@@ -27,7 +27,13 @@ runs:
       shell: bash
       run: |
         mkdir -p $HOME/.m2
-        cp ${{ inputs.maven-settings }} $HOME/.m2/settings.xml || true
+        if [ -e "${{ inputs.maven-settings }}" ]; then
+          echo Installing Maven settings file: ${{ inputs.maven-settings }}.
+          cp ${{ inputs.maven-settings }} $HOME/.m2/settings.xml
```

```suggestion
          cp "${{ inputs.maven-settings }}" $HOME/.m2/settings.xml
```

doubles quotes are always highly suggested around paths to avoid argument splitting if a space is present